tests/unit/test_loader.py

```diff
@@ -259,7 +259,7 @@ def testDefaultConfig(self):
                 'b': '4',
                 'c': '3'
             },
-            'mergeduplicateoptions': True
+            'mergeduplicateoptions': False
```

Comment:
I thought this should be `False` since the resulting config we compare it against has `'b': ['4', '2']` rather than overwriting the default value of `'4'`. Let me know if I misunderstood something here!